pkg/virt-controller/watch/vm_test.go

```diff
@@ -224,6 +224,12 @@ var _ = Describe("VirtualMachine", func() {
 			shouldExpectDataVolumeCreationPriorityClass(uid, labels, annotations, "", idx)
 		}
 
+		shouldFailDataVolumeCreationNoResourceFound := func() {
+			cdiClient.Fake.PrependReactor("create", "datavolumes", func(action testing.Action) (handled bool, obj runtime.Object, err error) {
+				return true, nil, fmt.Errorf("the server could not find the requested resource (post datavolumes.cdi.kubevirt.io)")
```

Comment:
Is this what the "dummyinformer" returns in case CDI is not installed?

Reply:
yep I ran the situation manually

Reply:
This is not `dummyInformer`, this is the real client that is always there.

Reply:
Yeah I just wanted to know this simulates the real thing, where if CDI doesn't exist the dummyinformer is used